powerauth-webflow-client/pom.xml

```diff
@@ -92,8 +74,16 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
```

Comment:
Do we really need `h2` dependency in the _compile_ scope?

Reply:
We can remove `h2` dependency completely, the client should use in memory repositories after migration from `spring-social` to `spring-security-oauth2-client`.